.drone.starlark

```diff
@@ -601,6 +601,418 @@ config = {
+			'suites': {
+				'webUICore1': 'webUICore1-enc-UK',
+				'webUICore2': 'webUICore2-enc-UK',
+			},
+			'databases': [
+				'mysql:5.7',
+			],
+			'servers': [
+				'latest',
+			],
+			'phpVersions': [
+				'7.1',
+			],
+			'runCoreTests': True,
+			'federatedServerNeeded': True,
+			'extraApps': {
+				'encryption': ''
+			},
+			'cron': 'nightly',
+			'extraSetup': [
+				{
+					'name': 'configure-app',
+					'image': 'owncloudci/php:7.0',
+					'pull': 'always',
+					'commands': [
+						'cd /var/www/owncloud/server',
+						'bash ./apps/user_ldap/tests/acceptance/setConfig.sh',
+						'php occ ldap:show-config',
+						'php occ ldap:test-config "LDAPTestId"',
+						'php occ user:sync "OCA\User_LDAP\User_Proxy" -m remove',
+						'php occ user:list',
+					]
+				},
+				{
+					'name': 'configure-app-on-federated-server',
+					'image': 'owncloudci/php:7.0',
+					'pull': 'always',
+					'commands': [
+						'cd /var/www/owncloud/federated',
+						'php occ market:install user_ldap',
+						'bash /var/www/owncloud/server/apps/user_ldap/tests/acceptance/setConfig.sh',
+						'php occ ldap:show-config',
+						'php occ ldap:test-config "LDAPTestId"',
+						'php occ user:sync "OCA\User_LDAP\User_Proxy" -m remove',
+						'php occ user:list',
+					]
+				},
+				{
+					'name': 'configure-encryption',
+					'image': 'owncloudci/php:7.1',
+					'pull': 'always',
+					'commands': [
+						'cd /var/www/owncloud/server',
+						'php occ encryption:enable',
+						'php occ encryption:select-encryption-type user-keys --yes',
+						'php occ config:list'
+					]
+				}
+			],
+		},
+		'core-api-acceptance-encryption-masterkey-nightly': {
+			'suites': {
+				'apiAuth': 'apiAuth-enc-MK',
+				'apiAuthOcs': 'apiAuthOcs-enc-MK',
+				'apiCapabilities': 'apiCapabilities-enc-MK',
+				'apiComments': 'apiComments-enc-MK',
+				'apiFavorites': 'apiFavorites-enc-MK',
+				'apiMain': 'apiMain-enc-MK',
+				'apiSharees': 'apiSharees-enc-MK',
+				'apiShareManagement': 'apiShareManagement-enc-MK',
+				'apiShareManagementBasic': 'apiShareMgmtBasic-enc-MK',
+				'apiShareOperations': 'apiShareOperations-enc-MK',
+				'apiShareReshare': 'apiShareReshare-enc-MK',
+				'apiShareUpdate': 'apiShareUpdate-enc-MK',
+				'apiSharingNotifications': 'apiSharingNot-enc-MK',
+				'apiTags': 'apiTags-enc-MK',
+				'apiTrashbin': 'apiTrashbin-enc-MK',
+				'apiVersions': 'apiVersions-enc-MK',
+				'apiWebdavLocks': 'apiWebdavLocks-enc-MK',
+				'apiWebdavLocks2': 'apiWebdavLocks2-enc-MK',
+				'apiWebdavMove': 'apiWebdavMove-enc-MK',
+				'apiWebdavOperations': 'apiWebdavOperations-enc-MK',
+				'apiWebdavProperties': 'apiWebdavProperties-enc-MK',
+				'apiWebdavUpload': 'apiWebdavUpload-enc-MK',
+			},
+			'databases': [
+				'mysql:5.7',
+			],
+			'servers': [
+				'latest',
+			],
+			'phpVersions': [
+				'7.1',
+			],
+			'runCoreTests': True,
+			'cron': 'nightly',
+			'extraApps': {
+				'encryption': ''
+			},
+			'extraSetup': [
```

Comment:
IMO this will also need the `configure-app` `extraSetup` block like in `core-api-federation-encryption-userkeys-nightly` because it will not get added from the `extraSetup` in `defaults`